### PR TITLE
get block by hash

### PR DESCRIPTION
**Pull request by stormeye2000**

### CODE REVIEW

File: test/features/steps/utils/exec.py [hunk 1]
```diff
@@ -11,3 +11,9 @@ def get_latest_block(self):
                                                os.popen(
                                                    "docker exec -t storm-nctl /bin/bash -c 'source "
                                                    "casper-node/utils/nctl/sh/views/view_chain_block.sh'").read()))
+
+    def get_latest_block_by_param(self, params):
+        return json.loads(self.clean_input.sub('',
+                                               os.popen(
+                                                   "docker exec -t storm-nctl /bin/bash -c 'source "
```
Comment (meywood) [inline test/features/steps/utils/exec.py:18]: shouldn't we get getting the 'storm-nctl' from some test properties?

File: test/features/steps/blocks.py
Comment (meywood) [inline test/features/steps/blocks.py:1]: Should really have a comment of the file describing the feature

File: test/features/steps/utils/exec.py [hunk 2]
```diff
@@ -11,3 +11,9 @@ def get_latest_block(self):
                                                os.popen(
                                                    "docker exec -t storm-nctl /bin/bash -c 'source "
                                                    "casper-node/utils/nctl/sh/views/view_chain_block.sh'").read()))
+
+    def get_latest_block_by_param(self, params):
+        return json.loads(self.clean_input.sub('',
```
Comment (meywood) [inline test/features/steps/utils/exec.py:16]: Duplicate code block here create a method the takes the parameter differences for get_latest_block and get_latest_block_by_param

**Review by meywood — CHANGES_REQUESTED** [review]

File: test/features/steps/utils/exec.py (hunk 1 above)
Comment (stormeye2000) [reply test/features/steps/utils/exec.py:18]: yes I plan to tidy this up next commit

**Review by meywood — APPROVED** [review]
Requested changes to be addressed in next PR

**Pull request merged**